src/cli/cli_link_metrics.cpp

```diff
@@ -242,25 +280,26 @@ template <> otError LinkMetrics::Process<Cmd("mgmt")>(Arg aArgs[])
             pLinkMetrics = nullptr;
         }
         /**
-         * @cli linkmetrics mgmt enhanced-ack register
+         * @cli linkmetrics config enhanced-ack register
          * @code
-         * linkmetrics mgmt fe80:0:0:0:3092:f334:1455:1ad2 enhanced-ack register qm
-         * Done
-         * > Received Link Metrics Management Response from: fe80:0:0:0:3092:f334:1455:1ad2
+         * linkmetrics config fe80:0:0:0:3092:f334:1455:1ad2 enhanced-ack register qm
+         * Received Link Metrics Management Response from: fe80:0:0:0:3092:f334:1455:1ad2
          * Status: Success
+         * Done
          * @endcode
          * @code
-         * > linkmetrics mgmt fe80:0:0:0:3092:f334:1455:1ad2 enhanced-ack register qm r
-         * Done
-         * > Received Link Metrics Management Response from: fe80:0:0:0:3092:f334:1455:1ad2
+         * > linkmetrics config fe80:0:0:0:3092:f334:1455:1ad2 enhanced-ack register qm r
+         * Received Link Metrics Management Response from: fe80:0:0:0:3092:f334:1455:1ad2
          * Status: Cannot support new series
+         * Done
          * @endcode
-         * @cparam linkmetrics mgmt @ca{peer-ipaddr} enhanced-ack register [@ca{qmr}][@ca{r}]
-         * [`q`, `m`, and `r`] map to #otLinkMetricsValues. Per spec 4.11.3.4.4.6, you can
-         * only use a maximum of two options at once, for example `q`, or `qm`.
-         * - `q`: Layer 2 LQI.
-         * - `m`: Link Margin.
-         * - `r`: RSSI.
+         * @cparam linkmetrics config [@ca{async}] @ca{peer-ipaddr} enhanced-ack register [@ca{qmr}][@ca{r}]
+         * - `async`: Use the non-blocking mode.
+         * - [`q`, `m`, and `r`] map to #otLinkMetricsValues. Per spec 4.11.3.4.4.6, you can
+         *   only use a maximum of two options at once, for example `q`, or `qm`.
+         *   - `q`: Layer 2 LQI.
+         *   - `m`: Link Margin.
+         *   - `r`: RSSI.
          * .
```

Comment:
```suggestion
         * @par
```

Needs to start a new paragraph here so that it imports properly.

Reply:
Updated